composer.json

```
@@ -13,8 +13,9 @@
         }
     ],
     "require": {
-        "php": ">=7.4.0",
-        "ronasit/laravel-helpers": "^2.1"
+        "php": "^8.3",
+        "laravel/framework": "^11.21",
```

let's use 30 minor version 

```suggestion
        "laravel/framework": "^11.30",
```